### PR TITLE
Added qr generator demo

### LINKED ISSUE #12: Find and investigate how to generate QR codes from text
**Is your feature request related to a problem? Please describe.**
We need a Python library that accepts text and generate QR codes (images)

**Describe the solution you'd like**
TODO:
* [x] Simple example of usage (demo)
* [x] Come up with the info to be stored in the QR code

**Additional context**
* Employee first and last names are required to be depicted on the QR code
* See the ref https://pypi.org/project/qrcode/

### PR DESCRIPTION
**Pull request by MykytaKyt**
# What does this PR do?

Adding demo qr generator
## Issue reference

Fixes #12 

## Explanation

This script generates qr code from the csv file

### CODE REVIEW

File: examples/qr-generator-demo.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import pandas as pd
```
Comment (aartiukh) [inline examples/qr-generator-demo.py]: Please create `qr-generator-demo` directory and move all these files there.

File: examples/qr-generator-demo.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+import qrcode
+
+
+def image_generator(csv_file_path):
+    df = pd.read_csv(csv_file_path, usecols=['employee_name', 'equipment_name', 'equipment_id'])
+    for i in range(len(df)):
+        df1=df.loc[i, :]
```
Comment (aartiukh) [inline examples/qr-generator-demo.py]: We can remove `df1` and use `df.loc[i, :]` below

File: examples/qr-generator-demo.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+        df_list = df1.values.tolist()
+        df_list = ', '.join(map(str, df_list))
+        qr = qrcode.QRCode()
+        qr.add_data(df_list)
+        qr.make()
+        img = qr.make_image()
+        file_name = df["employee_name"][i]
+        img.save(file_name+'.png')
```
Comment (aartiukh) [inline examples/qr-generator-demo.py]:
```suggestion
        img.save(file_name + '.png')
```

File: examples/qr-generator-demo.py [hunk 4]
```diff
@@ -0,0 +1,20 @@
+import pandas as pd
+import qrcode
+
+
+def image_generator(csv_file_path):
+    df = pd.read_csv(csv_file_path, usecols=['employee_name', 'equipment_name', 'equipment_id'])
+    for i in range(len(df)):
```
Comment (aartiukh) [inline examples/qr-generator-demo.py]: add an empty line before loop

File: examples/qr-generator-demo.py [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+        df1=df.loc[i, :]
+        df_list = df1.values.tolist()
+        df_list = ', '.join(map(str, df_list))
+        qr = qrcode.QRCode()
+        qr.add_data(df_list)
+        qr.make()
```
Comment (aartiukh) [inline examples/qr-generator-demo.py]: add an empty line after this string

File: examples/qr-generator-demo.py [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+        img = qr.make_image()
+        file_name = df["employee_name"][i]
+        img.save(file_name+'.png')
+
+if __name__ == "__main__":
+    csv_file_path = "test_data.csv"
+    image_generator(csv_file_path)
```
Comment (aartiukh) [inline examples/qr-generator-demo.py]: Add an empty line at the end of file

**Review by aartiukh — CHANGES_REQUESTED** [review]

File: examples/qr-generator-demo.py (hunk 1 above)
Comment (MykytaKyt) [reply examples/qr-generator-demo.py]: done

**Review by aartiukh — APPROVED** [review]
LGTM

**Review by OleksandrZhytnyk — APPROVED** [review]

**Review by paulzcooper — APPROVED** [review]

**Pull request merged**